config/default.yml

```diff
@@ -496,6 +496,13 @@ Rails/UniqBeforePluck:
     - aggressive
   AutoCorrect: false
 
+Rails/UniqueValidationWithoutIndex:
+  Description: 'Uniqueness validation should be with a unique index.'
+  Enabled: true
```

Comment:
Should be the status `pending`, or not?

Reply:
RuboCop core has not yet released cop with `pending`. Here, let's try with `true` first.